src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -614,80 +582,6 @@ class spell_dk_death_strike : public SpellScriptLoader
         }
 };
 
-// 85948 - Festering Strike
-class spell_dk_festering_strike : public SpellScriptLoader
-{
-    public:
-        spell_dk_festering_strike() : SpellScriptLoader("spell_dk_festering_strike") { }
-
-        class spell_dk_festering_strike_SpellScript : public SpellScript
-        {
-            PrepareSpellScript(spell_dk_festering_strike_SpellScript);
-
-            bool Validate(SpellInfo const* /*spellInfo*/) override
-            {
-                return ValidateSpellInfo({ SPELL_DK_FESTERING_WOUND });
-            }
-
-            void HandleScriptEffect(SpellEffIndex /*effIndex*/)
-            {
-                GetCaster()->CastCustomSpell(SPELL_DK_FESTERING_WOUND, SPELLVALUE_AURA_STACK, GetEffectValue(), GetHitUnit(), TRIGGERED_FULL_MASK);
-            }
-
-            void Register() override
-            {
-                OnEffectHitTarget += SpellEffectFn(spell_dk_festering_strike_SpellScript::HandleScriptEffect, EFFECT_2, SPELL_EFFECT_DUMMY);
```

Comment:
This script is still correct, just needs swapping EFFECT_2 to EFFECT_1 in `Register()`